### ISSUE TITLE
dronekit support for python 3.x

### ISSUE THREAD

**Issue by Gourav1997** [dronekit/dronekit-python#1001]
Is the dronekit available for python 3.x as python 2.x has been depreciated because in the 3DR dronekit page it still says that that python 3.0 is not supported. Please help asap.

**Comment by pietrodn** [comment]
The latest release fully supports Python 3. The website is outdated 😊

**Comment by gaetanokent** [comment]
@pietrodn thanks for the info. That means, if I install dronekit using "sudo pip install dronekit" I get a version that runs with python3. correct? Or should I specify a certain version of dronekit? Thanks a lot

**Comment by pietrodn** [comment]
Correct, the current PyPI version is Python 3 compatible. Pip is enough 👌
